tensorboard/webapp/metrics/views/main_view/card_grid_component.scss

```diff
@@ -48,7 +48,6 @@ card-view {
   @include tb-theme-foreground-prop(border, border, 1px solid);
   border-radius: 4px;
   box-sizing: border-box;
-  contain: layout paint;
```

Comment:
Because modal components (like the column header context menus) are defined as descendants of card-view, it's impossible to make them show on top of the next card-view, which has its own overriding [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context):
![context_menu_covered](https://github.com/tensorflow/tensorboard/assets/736199/b5216a90-2a26-4312-919f-f00e18cdcc39)

Removing the stacking context seems the only way to solve the modal z-index issue without re-implementing the modal to render in the overlay container, but I'm a bit concerned about the performance implications of removing `contain: paint` - IIUC, pages with many open panels with many line charts may experience some additional render lag?

@bmd3k I see that you last worked on this part of the code - do you happen to have any insight into the safety of removing `contain: paint`? FWIW there seems to be no noticeable difference when testing with my limited data, but just want to double check.

Reply:
Ok after additional experimentation, I'm nearly certain that removing `contain: paint` is a very bad idea - it makes interactions like opening/closing the settings menu much more clunky even with my small dataset. I'll remove the change from this PR.

This likely means that I'll have to reimplement the context menu to prevent it being covered in the minimized table views. I hope this is minor enough to not be considered a blocker :'(